app/models/shipit/repository.rb

```diff
@@ -14,6 +44,16 @@ class Repository < ApplicationRecord
     validates :name, format: { with: /\A[a-z0-9_\-\.]+\z/ }, length: { maximum: NAME_MAX_SIZE }
 
     has_many :stacks, dependent: :destroy
+    has_many :review_stacks, dependent: :destroy
+
+    enum(
+      provisioning_behavior: {
+        allow_all: 0,
```

Comment:
We don't use numeric enums, they're a big PITA long term. See: https://github.com/Shopify/shipit-engine/blob/329272fb79ea1496a004b62a470abdf076d12c57/app/models/shipit/status.rb#L7-L8

Reply:
> they're a big PITA long term

...and there's a part of me that knew this. Thanks for the feedback. We wrote this almost a year ago, I'll dig into the context, but I doubt there was a pressing architectural need for a numeric enum. I suspect a change for this will be inbound shortly.

Reply:
Addressed by: 33c80107